test/integration/test-form-get-length-sync.js

```diff
@@ -74,3 +75,30 @@ var fs = require('fs');
 
   assert.equal(expectedLength, calculatedLength);
 })();
+
+(function testReadableStreamData() {
+  var form = new FormData();
+  var expectedLength = 0;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['expectedLength' is defined but never used.](https://www.codacy.com/app/form-data/form-data/file/875565512/issues/source?bid=2805153&fileBranchId=2805172#l81)